packages/react/src/components/Tabs/Tab/Tab.module.css

```diff
@@ -0,0 +1,82 @@
+.tabItem {
+  --fdsc-icon-size: var(--fds-sizing-4);
+  --fdsc-typography-font-family: inherit;
+  --fdsc-bottom-border-color: transparent;
+
+  display: flex;
+  flex-direction: row;
+  box-sizing: border-box;
+  gap: var(--fds-spacing-1);
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  padding: var(--fds-spacing-2) var(--fds-spacing-3);
+  border: none;
+  border-radius: 0;
+  background-color: transparent;
+  cursor: pointer;
+  color: var(--fds-semantic-text-neutral-subtle);
+  position: relative;
+}
+
+.icon > svg {
+  display: flex;
+  height: 1.75em;
+  width: auto;
+}
+
+.tabItem.small {
+  --fdsc-icon-size: var(--fds-sizing-5);
+
+  font: var(--fds-typography-interactive-small);
+  font-family: var(--fdsc-typography-font-family);
+  padding: var(--fds-spacing-2) var(--fds-spacing-4);
+}
+
+.tabItem.medium {
+  --fdsc-icon-size: var(--fds-sizing-6);
+
+  font: var(--fds-typography-interactive-medium);
+  font-family: var(--fdsc-typography-font-family);
+  padding: var(--fds-spacing-3) var(--fds-spacing-5);
+}
+
+.tabItem.large {
+  --fdsc-icon-size: var(--fds-sizing-7);
+
+  font: var(--fds-typography-interactive-large);
+  font-family: var(--fdsc-typography-font-family);
+  padding: var(--fds-spacing-4) var(--fds-spacing-6);
+}
+
+.tabItem:hover {
+  --fdsc-bottom-border-color: var(--fds-semantic-border-neutral-subtle);
+
+  color: var(--fds-semantic-text-neutral-default);
+}
```

Comment:
We forgot to add the media query to check if hover is available here.

Reply:
Yes, right! Nice catch 🙂